components/Footer/src/index.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import BaseProps from '@gemeente-denhaag/baseprops';
+import ResponsiveContent from '/components/ResponsiveContent';
+
+import './index.scss';
+
+export interface FooterListItemData {
+  label: string;
+  url: string;
+}
+
+export interface FooterLegalProps extends Omit<BaseProps, 'tabIndex' | 'classNamees' | 'children'> {
```

Comment:
typo:`classNames`

Reply:
Why do we want to Omit className? I think maybe we could use the className as one of the arguments of `clsx` instead. If somehow somewhere we then need a tiny addition or change we can just add an extra className to the `<FooterLegalProps className="fancy-footer-snowflake"/>` ?